### ISSUE TITLE
[Style] Activity: Allow to style SDL on Repeat While loop.

### ISSUE THREAD

**Issue by The-Lum** [plantuml/plantuml#1707]
## Context
Observed on:
- [QA-18743](https://forum.plantuml.net/18743/how-to-color-the-shapes-of-a-repeat-while)

And to follow:
- #1423

## Wanted feature
- Could you allow style on SDL shape on Repeat While loop?

## Example

```puml
@startuml
<style>
.input {
  BackgroundColor  #ee1100
}
.output {
  BackgroundColor  #ff4422
}
.procedure {
  BackgroundColor  #feae2d
}
.load {
  BackgroundColor  #69d025
}
.save {
  BackgroundColor  #12bdb9
}
.continuous {
  BackgroundColor #3311bb
}
.task {
  BackgroundColor  #442299
}
</style>

start
  repeat :Input; <<input>>
  :Input; <<input>>
  repeat while (Successful?) is (NO)
  ->YES;
  repeat :Output; <<output>>
  :Output; <<output>>
  repeat while (Successful?) is (NO)
  ->YES;
  repeat :Procedure; <<procedure>>
  :Procedure; <<procedure>>
  repeat while (Successful?) is (NO)
  ->YES;
  repeat :Load; <<load>>
  :Load; <<load>>
  repeat while (Successful?) is (NO)
  ->YES;
  repeat :Save; <<save>>
  :Save; <<save>>
  repeat while (Successful?) is (NO)
  ->YES;
  repeat :Continuous; <<continuous>>
  :Continuous; <<continuous>>
  repeat while (Successful?) is (NO)
  ->YES;
  repeat :Task; <<task>>
  :Task; <<task>>
  repeat while (Successful?) is (NO)
  ->YES;
end
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/dPDDIyD048Rl-oiMYj0UZCas5cf3B1OFWbYXNeHUDjc9XiRTiXyL4V-xEzrQXJO5SvrzU5uoysPgBDVMlJKaC_QZWPn4jLmxIpy9fJUyNBrevQIOgKPfIdi0IHB7v8j4ojbshAhIb374rbgL89o6DX8uC85aexXeWKPZ4RDBX0pVj9aILeXYZ52ffAsbKyuSHNl3OP8K1PAMcrMB3ZzWZBhiOhSOibsLXpMiWLjwTOUBcj0issuip_tJiTaEVt-j6w3zXIjBCAPopVM0reRs7-O33vtdpxUBoH_vVBjVD8LD1_tHQRU0nv_Be6r_fX1pugrRsBq_Bhhmo27YOD9DlF25G0qM8uWF9jt4its9KFPRgH1oyhLRu9Fl8igmao7aOF8VCKX1IAMK1KsNCebOpD8eBS3oqLAURK2RMid-W4mzw7__Rm00)](https://www.planttext.com/?text=dPDDIyD048Rl-oiMYj0UZCas5cf3B1OFWbYXNeHUDjc9XiRTiXyL4V-xEzrQXJO5SvrzU5uoysPgBDVMlJKaC_QZWPn4jLmxIpy9fJUyNBrevQIOgKPfIdi0IHB7v8j4ojbshAhIb374rbgL89o6DX8uC85aexXeWKPZ4RDBX0pVj9aILeXYZ52ffAsbKyuSHNl3OP8K1PAMcrMB3ZzWZBhiOhSOibsLXpMiWLjwTOUBcj0issuip_tJiTaEVt-j6w3zXIjBCAPopVM0reRs7-O33vtdpxUBoH_vVBjVD8LD1_tHQRU0nv_Be6r_fX1pugrRsBq_Bhhmo27YOD9DlF25G0qM8uWF9jt4its9KFPRgH1oyhLRu9Fl8igmao7aOF8VCKX1IAMK1KsNCebOpD8eBS3oqLAURK2RMid-W4mzw7__Rm00) 

Regards,
Th.

**Comment by sohmuijai** [comment]
+1 please

**Comment by The-Lum** [comment]
FYI:
- That is fixed on [V1.2024.6](https://github.com/plantuml/plantuml/releases/tag/v1.2024.6) ✔️ 
- With minor regression _(fixed on next V1.2024.7 ✔️)_:
  - https://github.com/plantuml/plantuml/issues/1531#issuecomment-2239501010

Thanks for your works.
